python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1135,6 +1135,91 @@ def plot_gnss_cn0(self):
+        figure['layout']['yaxis2'].update(title="Degrees")
+
+        # Assign colors to each satellite.
+        data_by_sv = GNSSSatelliteMessage.group_by_sv(data)
+        svs_by_system = defaultdict(set)
+        indices_by_system = defaultdict(list)
+        svs = sorted(list(data_by_sv.keys()))
+        color_by_sv = self._assign_colors(svs)
+
+        for i, sv in enumerate(svs[::-1]):
+            name = satellite_to_string(sv, short=False)
+            system = get_system(sv)
+            sv_data = data_by_sv[sv]
+            svs_by_system[system].add(sv)
+
+            az_deg = sv_data['azimuth_deg']
+            el_deg = sv_data['elevation_deg']
+
+            time = sv_data['p1_time'] - float(self.t0)
+
+            # Plot the data. We set styles for coloring by SV.
+            text = ["P1: %.3f sec" % (t + float(self.t0)) for t in time]
+            figure.add_trace(go.Scattergl(x=time, y=az_deg, text=text,
+                                            name=name, hoverlabel={'namelength': -1},
+                                            mode='markers',
+                                            marker={'color': color_by_sv[sv], 'symbol': 'circle', 'size': 8},
+                                            showlegend=True,
+                                            legendgroup=name),
+                                1, 1)
+            indices_by_system[system].append(len(figure.data) - 1)
+            figure.add_trace(go.Scattergl(x=time, y=el_deg, text=text,
+                                            name=name, hoverlabel={'namelength': -1},
+                                            mode='markers',
+                                            marker={'color': color_by_sv[sv], 'symbol': 'circle', 'size': 8},
+                                            showlegend=False,
+                                            legendgroup=name),
+                                3, 1)
+            indices_by_system[system].append(len(figure.data) - 1)
+
+        # Add signal type selection buttons.
+        num_traces = len(figure.data)
+        buttons = [dict(label='All', method='restyle', args=['visible', [True] * num_traces])]
+        for system, indices in sorted(indices_by_system.items()):
+            if len(indices) == 0:
+                continue
+            visible = np.full((num_traces,), False)
+            visible[indices] = True
+            buttons.append(dict(label=f'{str(system)} ({len(svs_by_system[system])})', method='restyle',
+                                args=['visible', visible]))
+        figure['layout']['updatemenus'] = [{
+            'type': 'buttons',
+            'direction': 'left',
+            'buttons': buttons,
+            'x': 0.0,
+            'xanchor': 'left',
+            'y': 1.1,
+            'yanchor': 'top'
+        }]
+
+        self._add_figure(name='satellite_azimuth_elevation', figure=figure, title='Satellite Azimuth & Elevation')
```

Comment:
Title and filename should start with "GNSS" (or "gnss_") so it sorts with the other GNSS plots alphabetically, and add "Vs. Time" to the end similar to other time series plots